### DISCUSSION TITLE
以前に所属していた Organization で投稿した記事を、脱退した後で更新した場合の挙動について

### DISCUSSION THREAD

**Discussion by tfandkusu** [increments/qiita-discussions#463] [要望・アイデア・不具合]
不具合報告です。

# 気になったところ

以下の操作をするとエラーメッセージなどが表示されず記事を更新できない

- Organization A 所属中に記事Bを投稿。
- 転職に伴い Organization A から脱退し Organization C に所属。
- 記事Bを更新。「Organizationの紐付け」を Organization A にして「更新する」ボタンを押す。
- エラーメッセージなどが表示されず先に進まない。
- 「Organizationの紐付け」を Organization C にして「更新する」ボタンを押すと更新できる。

![スクリーンショット 2023-04-16 18 45 20](https://user-images.githubusercontent.com/16898831/232290842-2049be59-477f-40ed-96e1-ab885a4f6ca4.png)

# 期待する動作

## 案1

「Organizationの紐付け」に脱退した Organization は表示されない。

## 案2

「Organizationの紐付け」に脱退した Organization を設定して「更新する」ボタンを押した場合は、それは出来ないことが分かるエラーメッセージを表示する。

## 案3

「Organizationの紐付け」に Organization 所属中に新規投稿した記事に関しては、更新時に脱退した Organization を設定可能にする。

**Comment by tanaka-asami415** [comment]
@tfandkusu 
ご報告いただきありがとうございます🙇
ご不便をおかけして申し訳ありません。こちら修正いたします。

**Comment by kimkim0814** [comment]
@tfandkusu
お待たせいたしました！

動作として、案3になるように修正いたしました！
すでに脱退したOrganizationが紐づいている記事は、
そのOrganizationに紐づけたまま更新することができます😊
